spec/index.bs

```diff
@@ -7144,8 +7144,8 @@ dictionary GPURenderPassDepthStencilAttachment {
 
     - |this|.{{GPURenderPassDepthStencilAttachment/view}} must have a [=depth or stencil renderable
         format=].
-    - |this|.{{GPURenderPassDepthStencilAttachment/view}} must be a view of a
-        single [=texture subresource=].
+    - |this|.{{GPURenderPassDepthStencilAttachment/view}} must be a view of
+         all the [=texture subresources=] in the texture.
```

Comment:
Isn't it all aspects, but still only one array layer?

Reply:
fixed